src/pages/docs_sdk_forgot-password.html

```diff
@@ -0,0 +1,44 @@
+<div class="section">
+  <h2><a href="/docs/">Docs</a> : <a href="/docs/sdk/">SDK</a> : forgotPassword</h2>
+
+  <p><span class="font-semibold">forgotPassword</span> lets you send a temporary password to the user's email address. This API will return a promise that gets resolved once the email with the temporary password has been sent.</p>
```

Comment:
Just commenting here so we don't forget

We also mentioned adding a temporary password flag in the response to signIn()